### PR TITLE
Fixes "Unused Imports" issues from the Codacy.

### PR DESCRIPTION
**Pull request by pimts**
Fixes "Unused Imports" issues from the Codacy.

See https://github.com/EduMIPS64/edumips64/issues/222

### CODE REVIEW

File: src/main/java/org/edumips64/core/is/FPArithmeticInstructions.java [hunk 1]
```diff
@@ -24,10 +24,19 @@
...
+import org.edumips64.core.IrregularWriteOperationException;
+import org.edumips64.core.MemoryElementNotFoundException;
+import org.edumips64.core.fpu.FPDivideByZeroException;
+import org.edumips64.core.fpu.FPInstructionUtils;
+import org.edumips64.core.fpu.FPInvalidOperationException;
+import org.edumips64.core.fpu.FPOverflowException;
+import org.edumips64.core.fpu.FPUnderflowException;
+import org.edumips64.core.fpu.RegisterFP;
 
+//per diagnostica
```
Comment (lupino3) [inline src/main/java/org/edumips64/core/is/FPArithmeticInstructions.java:39]: You can remove this comment, it looks obsolete.

**Comment by lupino3** [comment]
Hi, thanks a lot! The change looks good. There is an obsolete comment which you moved around: can you please delete it? Other than that, the change looks great and I'll merge it once you remove the comment.

If you prefer not to remove the comment, just let me know and I'll merge the change as it is.

Thanks for your contribution!

File: src/main/java/org/edumips64/core/is/FPArithmeticInstructions.java (hunk 1 above)
Comment (pimts) [reply src/main/java/org/edumips64/core/is/FPArithmeticInstructions.java:39]: Sure! No problem, shall I remove all occurrences of that comment across all classes?

**Comment by lupino3** [comment]
Thanks a lot @pimts for the change and for the clean-ups! Merging the change.

There is one check that's failing, but it's not failing because of your change.

**Pull request merged**